### DISCUSSION TITLE
Can VEROS simulate regional ocean models?

### DISCUSSION THREAD

**Discussion by q1165600785** [team-ocean/veros#635] [General]
Can Veros specify the area I want?
I want to use Veros to simulate the tidal process of the Bohai Sea and the Yellow Sea in China. Can Veros do it?

Is it consistent with the results of fvcom and poms?

**Comment by dionhaefner** [comment]
Yes, although it's not very convenient because you will have to create all regional inputs for your region of choice yourself. See https://github.com/team-ocean/veros/blob/main/veros/setups/north_atlantic/north_atlantic.py for a regional setup of the North Atlantic.

> Is it consistent with the results of fvcom and poms?

We do not evaluate Veros for consistency with other models.
